apiary.apib

```diff
@@ -275,6 +276,7 @@ This collection is paginated.
               "releases": [{
                 "version": "0.4.1",
                 "url": "https://hex.pm/api/packages/plug/releases/0.4.1",
+                "html_url": "https://hex.pm/packages/plug/0.4.1",
```

Comment:
Ah, that's right. We should not include it on the "minimal" view.

@wojtekmach We can merge this and fix after merging. Can you please fix on hexpm master also?